.github/workflows/tests.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
-      - run: pip install $(grep mypy== requirements-tests-py3.txt)
+      - run: pip install 'tomli>=1.1.0,<1.2.0' $(grep mypy== requirements-tests-py3.txt)
```

Comment:
We should add `tomli` to the requirements file instead of importing it manually here.